src/compiler/types.ts

```diff
@@ -6247,7 +6247,7 @@ namespace ts {
         Deferred = 7
     }
 
-    export const enum ScriptTarget {
+    export enum ScriptTarget {

```

Comment:
Leave this as `const` and just cast the ts namespac to `any` when doing the reverse mapping to silence the error about using a const enum (so `(ts as any).ScriptTarget[value]`). Don't worry, we preserve and un-const the const enums in our output, we do this elsewhere. 😊

Reply:
Done, thanks!